### DISCUSSION TITLE
Thanks for a great project & Complete units for sale

### DISCUSSION THREAD

**Discussion by asuweb** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#671] [General]
Firstly, big thanks to @visualapproach for this project and great guide to getting it all setup.  Works flawlessly.

Thanks also to @MacGyver1979 (#538) for the housing design.

Next project is to integrate it with Home Assistant to make best use of the solar.  

Happy to sell completed kits ready to install if anyone is interested.  

![20240208_162800](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/13250955/aa6d06ca-cc53-47f6-92fc-d39c0a571768)

![20240208_165748](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/13250955/57c6c353-34be-45e2-9200-4887e008dd98)

![20240209_122207](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/13250955/6a76e0b8-3039-46bd-80a4-458102fdd2da)

**Comment by the9kon** [comment]
@asuweb Are you still selling complete kits?

Thanks.

**Comment by Schimanski91** [comment]
Are You Still selling complete Kits?

**Comment by cth35** [comment]
Hello, I'm also interested if you still sells out of the box kit ;-)

**Comment by fbelsoe** [comment]
Hi, are you still selling the kit? Shipping to Austria?
Thanks a lot!
